src/pages/questionPage.js

```diff
@@ -6,31 +6,45 @@ import {
 import { createQuestionElement } from '../views/questionView.js';
 import { createAnswerElement } from '../views/answerView.js';
 import { quizData } from '../data.js';
+import { changeProgress } from '../views/progressBar.js';
+
 
 export const initQuestionPage = () => {
   const userInterface = document.getElementById(USER_INTERFACE_ID);
   userInterface.innerHTML = '';
 
   const currentQuestion = quizData.questions[quizData.currentQuestionIndex];
 
-  const questionElement = createQuestionElement(currentQuestion.text);
+  const questionElement = createQuestionElement(currentQuestion.text, quizData.currentQuestionIndex + 1);
 
   userInterface.appendChild(questionElement);
 
   const answersListElement = document.getElementById(ANSWERS_LIST_ID);
-
   for (const [key, answerText] of Object.entries(currentQuestion.answers)) {
     const answerElement = createAnswerElement(key, answerText);
     answersListElement.appendChild(answerElement);
+
   }
 
+// this code disabled and active button
+  document
```

Comment:
Why do you use `setAttribute('disabled', true)` and then remove the attribute in the next block?

Reply:
I wait code from Anastasia and leave on/off button logic in here, maybe need made comment

Reply:
Great thinking ahead!